lib/python/rosie/ws.py

```diff
@@ -99,72 +168,56 @@ def __init__(self, props, prefix, db_url):
         if source_url_node is not None:
             self.source_url = source_url_node.value
         self.dao = rosie.db.DAO(db_url)
+        self.root_service_url = root_service_url
+
+        # Arguments for API points and formatting
+        self.q_args = self.get_arguments("q")
+        self.s_args = self.get_arguments("s")
+        # Need to provide a default, else get MissingArgumentError w/ no arg:
+        self.format = self.get_argument("format", default=None)
 
     def __call__(self):
```

Comment:
I have absolutely no idea what function this was originally supposed to serve!

Reply:
Nor I. There were also ``self.exposed = True`` instance variables set but not used at all in the logic...

I will remove ``__call__`` given your backing to my suspicions that it was redundant. I should have investigated whether it was at some point.

Reply:
I've a hunch `self.exposed` might be related to the cherrypy `expose` decorator.